src/mesh/Router.cpp

```diff
@@ -339,41 +364,28 @@ Routing_Error perhapsEncode(MeshPacket *p)
             char compressed_out[Constants_DATA_PAYLOAD_LEN] = {0};
 
```

Comment:
This should be alright for most cases, but just allocate say 10% more for safety just in case the user types in as many bytes of text that will expand.  But I think this will not happen in real life, but only when unit testing edge cases.